CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Fix joining cgroup of instance started as root, with cgroups v1,
   non-default cgroupfs manager, and no device rules.
 - Ensure `DOCKER_HOST` is honored in non-build flows.
+- Remove unwanted colors to STDERR. #1259
```

Comment:
Please put this in a new `### Bug fixes` section, and don't reference the issue here since we don't usually do that.

Reply:
Also squash down to one commit.

Reply:
thanks! resolved in this commit
https://github.com/apptainer/apptainer/pull/1268/commits/d8ee12fd706ec6d809eeee9b643bcc087b719944